api/client.py

```diff
@@ -51,10 +54,13 @@ def listen(args):
     except KeyboardInterrupt as e:
         print(f"Stopping.")
     finally:
-        path = '/'.join(['unsubscribe', args.channel])
-        url = urllib.parse.urljoin(args.url, path)
-        res = requests.post(url, headers=headers)
-        res.raise_for_status()
+        try:
+            path = '/'.join(['unsubscribe', args.channel])
+            url = urllib.parse.urljoin(args.url, path)
+            res = requests.post(url, headers=headers)
+            res.raise_for_status()
+        except Exception as e:
+            print(f"Error: {e}")
```

Comment:
> I don't think we need a try-except block here as nothing is being done differently when catching the exception. It's important to catch the KeyboardInterrupt in the main loop to stop and unsubscribe, but then if unsubscribing fails there's nothing we can do about it other than print the exception - which is what happens if you don't catch it at all.

In case of unsubscribing, If we don't catch the exception, It ends up throwing it:
```
Traceback (most recent call last):
  File "/home/kernelci/api/client.py", line 94, in <module>
    args.func(args)
  File "/home/kernelci/api/client.py", line 71, in listen
    res.raise_for_status()
  File "/home/kernelci/.local/lib/python3.10/site-packages/requests/models.py", line 953, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 422 Client Error: Unprocessable Entity for url: http://localhost:8000/unsubscribe/node
```
That's why I added try-except block in finally.